resources/service.rb

```diff
@@ -0,0 +1,53 @@
+# To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+#
+# Author:: Corey Hemminger
```

Comment:
I do believe some credit should be given to me here, as a lot of this code looks copy/pasted from recipes that I developed in the original repo.